### PR TITLE
Replace default_scope with scopes for sorting

### PR DESCRIPTION
**Pull request by tistatos**
Since you cant override the default scope (even explicitly), I think we should avoid using it.
Instead I've added a scope to get both drinks and drink_types ordered by name

### CODE REVIEW

**Comment by tistatos** [comment]
:eyes:

File: app/views/drink_types/index.json.jbuilder [hunk 1]
```diff
@@ -3,7 +3,9 @@ json.drink_list(@drink_types) do |drink_type|
   json.extract! drink_type, :id, :name, :description
   json.url stock_url(drink_type)
   json.admin_url edit_drink_type_url(drink_type)
-  json.drinks drink_type.drinks.where(instock: true).select('name', 'country', 'price', 'slug', 'brewery') do |drink|
+  json.drinks drink_type.drinks.ordered_by_name.where(instock: true)
+    .select('name', 'country', 'price', 'slug', 'brewery') do |drink|
+
```
Comment (klaseskilson) [inline app/views/drink_types/index.json.jbuilder]: I think the readability here isn't 100%. Might I suggest that we pass the `select` params from a variable such as this:

```ruby
# ...
drink_params = %w(name country price slug brewery)
json.drinks drink_type.drinks.ordered_by_name.where(instock: true).select(drink_params) do |drink|
  # ...
end
```

**Comment by klaseskilson** [comment]
(This is btw the perfect sign that we need to improve our test coverage!)

**Comment by tistatos** [comment]
:white_check_mark: fixed variable for parameters

**Comment by klaseskilson** [comment]
😻! 🆗!  🚢 🇮🇹 !

**Pull request merged**